syntax = "proto3";

package user;

service UserService {
    rpc GetUser(UserRequest) returns (UserResponse);
}

message UserRequest {
    string user_id = 1;
}

message UserResponse {
    string user_id = 1;
    string user_name = 2;
    string user_email = 3;
}
